src/components/home/index.tsx

```diff
@@ -1,37 +1,61 @@
 import React from 'react'
 import { Address, EtherBalance, TokenBalance } from '../wallet/components'
+import { M2MPanel } from './components/Forms/M2MPanel'
 import { RedeemTokensPanel } from './components/Forms/RedeemTokensPanel'
-import { SendEtherForm } from './components/Forms/SendEtherForm'
-import { SendTokensForm } from './components/Forms/SendTokensForm'
 import LockedMxcTokens from './components/LockedMxcTokens'
 
 const Row = ({ caption, Content }: { caption: string; Content: any }) => (
   <tr>
     <td>
-      <h2>{caption}</h2>
+      <span className="t-s t-bold">{caption}</span>
     </td>
     <td>
       <Content />
     </td>
     <td />
   </tr>
 )
+const Heading = ({ caption, Content }: { caption: string; Content: any }) => (
+  <div>
+    <h1>{caption}</h1>
+    <h2>
+      <Content />
+      <i className="icon icon-contracts"/>
+    </h2>
+  </div>
+)
+
+const BtnAction = ({ caption, Content }: { caption: string; Content: any }) => (
+  <a href={caption}>
```

Comment:
`caption` is a bad property name in this case. `target` would be better, I find. Also you should not use anchors but the `Link` component from the router. Right now the whole page will reload when you click that link. Quite uncool.